ui_framework/src/components/form/check_box/check_box.test.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render, shallow } from 'enzyme';
+import { requiredProps } from '../../../test/required_props';
+import sinon from 'sinon';
+
+import {
+  KuiCheckBox,
+} from './check_box';
+
+describe('KuiCheckBox', () => {
+  test('renders', () => {
+    const component = (
+      <KuiCheckBox
+        onChange={()=>{}}
+        {...requiredProps}
+      />
+    );
+
+    expect(render(component)).toMatchSnapshot();
+  });
+
+  describe('Props', () => {
+    describe('isChecked', () => {
+      test('true renders checked', () => {
+        const component = (
+          <KuiCheckBox
+            isChecked={true}
```

Comment:
For boolean props, you don't need to pass in `{true}`. React will interpret the presence of `isChecked` as true, as long as you don't explicitly pass in `{false}`.

Reply:
Thanks!